Scripts/Runtime/Entities/Lifecycle/Migration/MigrationUtil.cs

```diff
@@ -113,23 +116,43 @@ public static void RegisterTypeForEntityPatching(Type type)
                 new TypeOffsetInfo(
                     entityOffsetStartIndex,
                     s_EntityOffsetList.Length));
-            
+
             //The size of the underlying data could have changed such that we re-allocated the memory, so we'll update
             //our shared statics
             UpdateSharedStatics();
         }
 
+        private static void ScanForCollections(string parentPathString, Type type)
+        {
+            //One of these fields might be a collection or contain a collection, best way to tell is to scan it and see if it has a pointer
+            FieldInfo[] fields = type.GetFields(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic);
+            foreach (FieldInfo field in fields)
+            {
+                Type fieldType = field.FieldType;
+                if (fieldType.IsPrimitive)
+                {
+                    continue;
+                }
+                if (fieldType.IsPointer)
```

Comment:
I don't know where you want to put this for future reference but we discussed how there may be cases where a pointer is being stored that we do not want to dive into an migrate. Maybe that memory address has already been migrated or something.

We want to figure out how to handle these situations. Maybe require developers to annotate unsafe fields with a `[RemapUnsafeEntityDataOnMove]` or something. Would need an `[ExcludeUnsafeEntityDataOnMove]` (names WIP)

Reply:
Cool, logged in #233